tests/scripts/thread-cert/pktverify/consts.py

```diff
@@ -52,12 +52,15 @@
 MAe1 = Ipv6Addr('fd0e::1234:777a:1')
 MAe2 = Ipv6Addr('::')
 MAe3 = Ipv6Addr('cafe::e0ff')
+MAS = [[Ipv6Addr(f"ff0e::{format(j, 'x')}:777a:{format(i, 'x')}") for j in range(16)] for i in range(16)]
```

Comment:
We can define it in tests/scripts/thread-cert/border_router/MATN_16_LargeNumberOfMulticastGroupSubscriptionsToBBR.py since it's only used by that test.

Reply:
I still prefer to keep it in consts.py. I found MA6 is only used in TC-21 but it's also defined in consts.py. In addition, the multicast addresses are defined in the 'general initial conditions' section of the doc, so it may be better to define the addresses in a common file.

Reply:
SGTM